html/MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include avocado_result_html/resources *
```

Comment:
I've actually tried to keep it inside `avocado/plugins` but Python/setuptools will try to package it automatically.  Keeping and optional package inside another doesn't seem to be how Python/distutils/setuptools was designed to work.

With regards to the name, it's just one directory name.  We may just as well move it to anything mode complex, say, `results-html-plugin`, but IMHO, that is so easily fixable (just a rename) if a clash *inside* the avocado source tree ever occur.  In the Python "site lib dir", it'll actually use the module name as defined in setuptools: `avocado_result_html`.